### DISCUSSION TITLE
[4.0] Media Manager - The account was not found

### DISCUSSION THREAD

**Discussion by pjdevries** [joomla/joomla-cms#34239] [Q&A] [answered]
What am I doing wrong when media manager shows error message "The account was not found", I don't see any of the available images and I can not create new folders or upload images?

**Comment by pjdevries** [comment, marked as answer]
That was mighty stupid. Apparently the media manager somehow opened in non existing folder "local-0".

**Comment by dgrammatiko** [comment]
For anyone facing this message: please log out from admin and re-login (or just clear the localsession data in your browser)

**Reply by pjdevries** [reply]
Thanx for the tip.

**Comment by sluc69** [comment]
I upgrade a backup version not secure (http) J4-beta7 to J4-rc1. All went OK as I describe in another topic.
Now I upgraded the secure (https) version and also get the  error message "The account was not found" when opening Media manager first time. Re-login give me access to all images as if the problem is solved.
But now I cannot DELETE, RENAME any media item! (Error deleting/renaming the item.)
